install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -356,3 +371,18 @@ rules:
       - get
       - list
       - watch
+
+---
+# Roles needed by cluster-api in order to read openshift-config:cloud-provider-config configmap
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: cloud-provider-configmap-reader
+  namespace: openshift-config
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmap
+    verbs:
+      - get
```

Comment:
Nit: missing new line at the end of the file

Should probably add `list` and `watch` to this too

Is the ConfigMap always named the same, or can it vary depending on cluster names etc? You _could_ be extra strict and restrict it to the particular ConfigMap name if that is statis

Reply:
Will do, and I it should always have the same name